python/dgl/mock_sparse/cpp_interface.py

```diff
@@ -0,0 +1,63 @@
+""" DGL sparse library C++ binding"""
+import os
+import torch
+from typing import List, Optional, Tuple
+
+package_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
+so_path = os.path.join(
+    package_path, "../tensoradapter/pytorch/libdgl_sparse.so"
+)
+torch.classes.load_library(so_path)
+
+_C_SparseMatrix = torch.classes.dgl_sparse.SparseMatrix
+
+
+# TODO Add docstring
+class SparseMatrix:
+    def __init__(self, c_sparse_matrix: _C_SparseMatrix):
+        self._c_sparse_matrix = c_sparse_matrix
+
+    @property
+    def val(self) -> torch.Tensor:
+        """Get the values of the nonzero elements.
+
+        Returns
+        -------
+        tensor
```

Comment:
`torch.Tensor`

Reply:
You can copy-paste the corresponding docstring from the mock-up library.

Reply:
Done.